### DISCUSSION TITLE
[BUG] Original_year set to 0 in deezer plugin and as-is

### DISCUSSION THREAD

**Discussion by arogl** [beetbox/beets#4459] [General]
I have noticed that the `original_year` is set to 0 when using the deezer plugin as well as an 'as-is' import.

In my config I do not have `original_date` set, so assume it has defaulted to no.

In the discogs plugin I notice `orginal_year` is explicitly set to `year` when no master recording is set.

Do we need to add similar lines to the other plugins?

Related to this is if I set the `orginal_year` in MP3 tag to match `year`, on a ```beet update``` the `year`, `month` and `date ` tags get set to 0.

**Comment by sampsyo** [comment]
0 is the default value, so that is what will happen if a backend doesn't provide an original year. Collecting that information from other backends would always be a useful feature!

The `original_date` config option causes `year` to contain the earlier release date, i.e., what would otherwise just go in `original_year`.
